app/README.md

```diff
@@ -1,60 +1,140 @@
-# radicle desktop client
+# App
 
-This is a cross-platform desktop client for Radicle.
-We currently support Linux and macOS.
+A frontend written in JavaScript using Svelte and Electron.
 
 
 ## Development
 
-The app is written in Svelte + Electron. For our dependency management and
-script execution we use `yarn`.
+For dependency management and script execution we use `yarn`. Code formatting
+is dictated by [prettier][pr] and enforced locally on a pre-commit basis with
+[husky][hu] and [lint-staged][ls].
 
-Code formatting is dictated by [prettier][pr] and we enforce it locally on
-a pre-commit basis with [husky][hu] and [lint-staged][ls].
 
+### Running the app
 
-### Setup
+1. Get the code: `git clone git@github.com:radicle-dev/radicle-upstream.git`.
+2. Install dependencies: `cd radicle-upstream/app && yarn install`.
+3. Start the app in development mode: `yarn start`.
 
-Run the app locally with hot reloading:
 
-1. Get the code: `git clone git@github.com:radicle-dev/radicle-upstream.git`
-2. Install dependencies: `cd radicle-upstream/app && yarn install`
-3. Start the app in development mode: `yarn start`
+### Running tests
 
+Before running tests locally you'll need to set up a test fixture repository:
+`git submodule foreach "git fetch --all"`.
 
-Build and package the app:
+- to run the tests do: `yarn test`
+- to troubleshoot tests in the Cypress GUI, run: `yarn test:debug`
 
-1. Build: `yarn dist`
-2. Get the generated package from: `dist/radicle-upstream-0.0.1.dmg`
+To isolate a single test for debugging purposes, use the `.only` method:
+```javascript
+context("a bazillion tests in this context", () => {
+  it("does one thing", () => {
+    ...
+  });
 
+  it.only("does another thing", () => {
+    ...
+  });
 
-#### Running tests locally
+  ...
+});
+```
+
+Then to execute just that one test, fire up the Cypress GUI: `yarn test:debug`
+and choose the respective `.spec.js` file from the list.
+
+**Note**: don't forget to remove all `.only` methods from tests before
```

Comment:
I wonder if you can add a hook or a lint to check that :thinking:

Reply:
Was also thinking about that, haven't found anything ready-made yet, hence the note.
An alternative would be to fail a build on CI immediately if it detects skipped tests.

Reply:
Looking forward to the day someone forgets :joy:

Reply:
🙈

Reply:
A bit out of scope for this issue, but we'll keep it in our TODOs: https://github.com/radicle-dev/radicle-upstream/issues/131